defaults/main/sysctl.yml

```diff
@@ -57,6 +45,23 @@ sysctl_settings:
   net.ipv6.conf.default.max_addresses: 1
   net.ipv6.conf.default.router_solicitations: 0
   net.ipv6.conf.default.use_tempaddr: 2
+
+regular_sysctl_settings:
```

Comment:
perhaps `generic_sysctl_settings` is better? I don't really now how regular they are.

Reply:
Yeah I was struggling to come up with a term there. Will do as suggested.